packages/turbo-workspaces/src/cli.ts

```diff
@@ -25,6 +25,11 @@ workspacesCli
     "Do not run a package manager install after conversion",
     false
   )
+  .option(
```

Comment:
I needed to make sure I was always getting a successful exit code for my scripts. This seemed like a reasonable flag to add, even beyond my specific use case.